src/SolidClient.js

```diff
@@ -185,6 +206,21 @@ class SolidClient {
     return accessUrl;
   }
 
+  isAboveVersion511(version) {
+    const versionNumber = version.split('/')[1];
+    const targetVersion = [5, 1, 1];
+    if (versionNumber) {
+      const versionNumbers = versionNumber.split('.');
+      for (let i = 0; i < targetVersion.length; i++) {
+        if (targetVersion[i] > versionNumbers[i]) {
+          return false;
+        }
+      }
+      return true;
+    }
+    return false;
+  }
```

Comment:
I wouldn't over-engineer and just `return /^solid-server\/5\.(1\.[1-9]|[2-9]|1\d)/.test(version);` (assuming that there will never be a NSS v6).